### PR TITLE
Feature/celo package

### PR DESCRIPTION
**Pull request by vlnevyhosteny**
Featuring celo sdk package.

### CODE REVIEW

File: packages/blockchain/celo/src/lib/services/celo.kms.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    //     !transactionConfig.gasPrice ||
+    //     transactionConfig.gasPrice === '0' ||
+    //     transactionConfig.gasPrice === 0 ||
+    //     transactionConfig.gasPrice === '0x0'
+    //   ) {
+    //     transactionConfig.gasPrice = await args.web3.getGasPriceInWei()
+    //   }
+    //   return (await client.eth.accounts.signTransaction(transactionConfig, fromPrivateKey as string))
+    //     .rawTransaction as string
+    // export const signKMSTransaction = async (tx: ChainTransactionKMS, fromPrivateKey: string, provider?: string) => {
```
Comment (vlnevyhosteny) [inline packages/blockchain/celo/src/lib/services/celo.kms.ts:32]: Is it possible to use same implementation as ETH? Or should I use implementation from `feature/submodule` which uses `CeloWallet` lib?

File: packages/blockchain/celo/src/lib/services/celo.web3.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import Web3 from 'web3'
+import { EvmBasedBlockchain, httpHelper, THIRD_PARTY_API } from '@tatumio/shared-core'
+
+export const celoWeb3 = (args: { blockchain: EvmBasedBlockchain }) => {
+  const evmBasedWeb3Result = evmBasedWeb3(args)
+
+  return {
+    ...evmBasedWeb3Result,
+    getClient(provider?: string, fromPrivateKey?: string): Web3 {
+      const web3 = evmBasedWeb3Result.getClient(provider)
```
Comment (vlnevyhosteny) [inline packages/blockchain/celo/src/lib/services/celo.web3.ts:11]: Not sure but I can use Web3 lib for CELO as well - https://docs.celo.org/developer-guide/contractkit/notes-web3-with-contractkit .

**Comment by vlnevyhosteny** [comment]
@npwork What about testing features like KMS or Web3? Is that something to focus on right now or we should do it later?

File: packages/blockchain/celo/src/lib/services/celo.kms.ts (hunk 1 above)
Comment (npwork) [reply packages/blockchain/celo/src/lib/services/celo.kms.ts:32]: I guess you need CeloWallet one

**Review by npwork — APPROVED** [review]

**Pull request merged**